pulseapi/entries/models.py

```diff
@@ -68,6 +68,12 @@ def with_related(self):
             'related_entry_creators__profile__related_user',
         )
 
+    def by_active_user(self):
+        """
+        Return all entries that have been created by pulse users who are set to "Active" by moderator.
+        """
+        return self.filter(published_by__is_active=True)
```

Comment:
**Issue** `published_by` points to the email user. We want to only show entries by user's with active profiles. The user accounts should all be active.